### PR TITLE
Added privacy policy for ruby-lang.org

### PR DESCRIPTION
**Pull request by hsbt**
Fixes https://github.com/ruby/www.ruby-lang.org/issues/3134

### CODE REVIEW

File: en/privacy/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This privacy policy covers ruby-lang.org.
+
+## Email
+
+We will not give away your email address to anyone, who is not related to the operations of ruby-lang.org. We will also never ask you to send us any of your passwords via email.
+
+## Logfiles
+
+ruby-lang.org record access logs of the requests that reach the web servers, but we use those files for issue investigation and statistical purposes.
```
Comment (sorah) [inline en/privacy/index.md:15]:
```suggestion
ruby-lang.org records access logs of the requests that reach the web servers, but we use those files for debugging and statistical purposes.
```

Comment (sorah) [inline en/privacy/index.md:15]: As w.r-l.o uses GitHub Pages, it's worth to mention https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#data-collection

> www.ruby-lang.org uses GitHub Pages in its backend thus it is subject to GitHub's Data Collection. Refer to https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#data-collection for GitHub's privacy statement.

File: en/privacy/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Cookies
+
+ruby-lang.org uses cookies to keep track of user preferences. Unless you login on the site, the cookies will not be used to store personal information and we do not give away the information from the cookies.
```
Comment (sorah) [inline en/privacy/index.md:19]: > ruby-lang.org uses cookies to keep track of user preferences.

Do we? I don't think w.r-l.o has a user preference like https://www.php.net/my.php.

```suggestion
Some sites under ruby-lang.org (e.g. bugs.ruby-lang.org) uses cookies to keep track of user preferences. Unless you login on the site, the cookies will not be used to store personal information and we do not give away the information from the cookies.
```

**Review by sorah — CHANGES_REQUESTED** [review]

File: en/privacy/index.md (hunk 2 above)
Comment (hsbt) [reply en/privacy/index.md:19]: Yes, I noticed this for bugs.ruby-lang.org, ml.ruby-lang.org. I'll pick your suggestion.

**Review by nurse — APPROVED** [review]

**Pull request merged**